File: API/Models/Persoon.cs
using System;
using System.Collections.Generic;

namespace MoneyGrip.Models
{
    public partial class Persoon
    {
        public Persoon()
        {
            Inkomst = new HashSet<Inkomst>();
        }

        public int Id { get; set; }
        public DateTime LaatstGewijzigd { get; set; }
        public string Voornaam { get; set; }
        public string Achternaam { get; set; }

        public ICollection<Inkomst> Inkomst { get; set; }
    }
}

File: Angular/src/app/personen/persoon/persoon.ts
import BasisBeheerOverzicht from '../../basisBeheerOverzicht'

export class Persoon extends BasisBeheerOverzicht {
  id: number;
  laatstGewijzigd: Date;
  voornaam: string;
  achternaam?: string;

  getValue(value: string) : any
  {
    switch(value)
    {
      case "Voornaam": return this.voornaam;
        break;
      case "Achternaam" : return this.achternaam;
        break;
      case "Id": return this.id;
        break;
      default: return "";
    }
  }
}
